runelite-client/src/main/java/net/runelite/client/plugins/barbarianassault/BarbarianAssaultPlugin.java

```diff
@@ -195,7 +177,7 @@ public void onVarbitChanged(VarbitChanged event)
 			{
 				overlay.setCurrentRound(null);
 
-				if (config.waveTimes() && gameTime != null)
+				if (config.waveTimes() && gameTime != null && client.isInInstancedRegion())
```

Comment:
Is this to try and avoid messages from the tutorials? How does that work as the tutorials are instanced?

Reply:
Yes, it is. The tutorial uses the IN_GAME_BA varbit also. 
After exiting a tutorial there is a 2-3 tick delay for the IN_GAME_BA varbit to change from 1 -> 0 (2-3 ticks of being on the mainland). 
When finishing a real wave the IN_GAME_BA varbit change occurs when it is still in the BA instance. Checking if your in an instance still stops the tutorial from announcing wave duration.

Reply:
This is the type of things comments were invented for.